test/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+add_subdirectory(util)
+
+file(GLOB NORMAL_TESTS "test_*.cpp")
+file(GLOB LARGE_TESTS "large_tests/*.cpp")
+file(GLOB FUZZY_TESTS "fuzz_*.cpp")
```

Comment:
(I know this is a CMake limitation, and I don't know if there is a good workaround.)